src/main/java/io/github/cottonmc/cotton/gui/widget/WButton.java

```diff
@@ -65,9 +72,14 @@ public WButton(Text label) {
 	 * @param label the label
 	 * @since 2.2.0
 	 */
-	public WButton(Icon icon, Text label) {
+	public WButton(@Nullable Icon icon, @Nullable Text label) {
 		this.icon = icon;
 		this.label = label;
+		this.height = BUTTON_HEIGHT;
+
+		if (this.icon != null && this.label != null) {
+			this.setAlignment(HorizontalAlignment.LEFT);
+		}
```

Comment:
```suggestion
```
The old behaviour is correct, the alignment can be changed by the user if needed. This is a breaking change to GUI layouts.

Reply:
okay this is correct.